### ISSUE TITLE
snarl.admin file to hide/disable icon

### ISSUE THREAD

**Issue by whizzling** [fullphat/snarl#48]
https://sites.google.com/site/snarlapp/help-and-support/snarl-administration-guide

Is this still accurate? I am trying to disable the icon/menu from being accessed but not having any success.

Any help appreciated, Thanks!

**Comment by fullphat** [comment]
Which version of Snarl are you using?

**Comment by whizzling** [comment]
Latest version r5.0-beta-5

**Comment by fullphat** [comment]
Ah, no, not yet.  I can add it pretty quickly though.  I’m planning to overhaul it all a bit too - nothing drastic though.  Are you just interested in hiding the app icon for now?

**Comment by fullphat** [comment]
Here’s the proposed approach for R5.0.  Some name changes and we use JSON now.  Most significant change is moving the global admin file to a web server rather than CIFS:

❏ Snarl sysadmin
    ❏ Look for local “sysadmin.redirect” 
    ❏ If exists, read “url” and grab “sysadmin.json” from it 
    ❏ Otherwise look for local “sysadmin.json” 
    ❏ Settings: “hide_icon” bool

**Comment by whizzling** [comment]
Yes, looking to hide the icon to prevent the user from editing or turning off the notifications. I am attempting to use the snarl as an alert system for our employees.

http://snarl.fullphat.net/concept-employee-messaging-tool/

**Comment by fullphat** [comment]
Excellent!  Keep me posted on progress please.  I’ll get the sysadmin stuff added and let you know when it’s released.

**Comment by fullphat** [comment]
Hi @whizzling - Beta 6 is now available (see post [here](https://github.com/fullphat/snarl/issues/49)) and adds the ability to control displaying the tray icon, amongst other things.  I haven't had change to update the docs just yet so drop me a note here if you'd like some help. 👍

**Comment by fruza** [comment]
Hi!. I'm also very interested in this feature (sysadmin.json). I've tried it and it works great!!. Another thing I'm also interested is in sysconfig.ssl with "target" so that I can centralize the settings for all snarls. Is it possible to set this en sysadmin.json or planned for the near future??

Thanks and regards.

Fernando.

**Comment by fullphat** [comment]
Hi Fernando!

Yes, you can do this now with R5.0 Beta 6. 😀 

Create a text file called `redirect.rc` in the same folder as `snarl.exe` and paste this into it:

    [targets]
    url={path}

Where `{path}` is the URL to your centralised settings file (e.g. `http://myserver/snarl/finance_team/admin.json`)

Check out the sysadmin guide here for more details: http://fullphat.github.io/snarl/sysadmin-guide

**Comment by fruza** [comment]
Many thanks!!, however it's no exactly what I need.

What I need is to place all of configuration files for snarl: %APPDATA%\full phat\snarl 5.0\etc in a centralized server and say to snarl where to look for them.

Regards, Fernando.

**Comment by fullphat** [comment]
Ok, I've added the ability to specify the json configuration file using a path instead.  Check out Beta 7! #51
